src/check/parse/IR.zig

```diff
@@ -1770,6 +1774,23 @@ pub const NodeStore = struct {
         header: HeaderIdx,
         statements: []const StatementIdx,
         region: Region,
+
+        pub fn toSExpr(self: @This(), gpa: Allocator, env: *base.ModuleEnv, ir: *IR) anyerror!sexpr.Node {
+            var file_node = try sexpr.Node.newNode(gpa, "file");
+
+            const header = ir.store.getHeader(self.header);
+            var header_node = try header.toSExpr(gpa, env, ir);
+
+            try file_node.appendNodeChild(gpa, &header_node);
+
+            for (self.statements) |stmt_id| {
+                const stmt = ir.store.getStatement(stmt_id);
+                var stmt_node = try stmt.toSExpr(gpa, env, ir);
+                try file_node.appendNodeChild(gpa, &stmt_node);
```

Comment:
As a general note (not for this PR, but a follow up), I think it would be reasonable to update the underly code to simply panic on any sort of allocation error. If we do that, I think all of the `try` go away across all the `toSExpr` functions.

Reply:
Done in 58efb0416a